src/controllers/badgeController.js

```diff
@@ -57,7 +57,7 @@ const badgeController = function (Badge) {
   };
 
   const postBadge = async function (req, res) {
-    if (!await hasPermission(req.body.requestor.role, 'createBadges')) {
+    if (!await hasPermission(req.body.requestor.role, 'createBadges') && !await hasIndividualPermission(req.body.requestor.requestorId, 'seeBadgeManagementTab')) {
```

Comment:
```suggestion
    if (!await hasPermission(req.body.requestor, 'createBadges') && !await hasPermission(req.body.requestor, 'seeBadgeManagementTab')) {
```
Probably need to pull the latest changes. The arguments have changed.